ibis/backends/tests/test_generic.py

```diff
@@ -1846,3 +1846,15 @@ def test_select_mutate_with_dict(backend):
 
     expr = t.select({"a": ibis.literal(1.0)}).limit(1)
     backend.assert_frame_equal(result, expected)
+
+
+@pytest.mark.broken(["mssql", "oracle"], reason="incorrect syntax")
+def test_isnull_equality(con, backend):
+    ibis.set_backend(con)
```

Comment:
This has to be monkeypatched in to avoid altering the default backend for other tests

Reply:
ARGH

I need to write a pre-commit check for this